source/database/mssql/scripts/dbscripts/PSP_PIMS_S47_00/Alter Up/002_PSP_PIMS_Alter_Up.sql

```diff
@@ -0,0 +1,2374 @@
+GO
+DROP TRIGGER IF EXISTS [dbo].[PIMS_RSCNOT_I_S_I_TR]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop trigger dbo.PIMS_RSCNOT_A_S_IUD_TR
+PRINT N'Drop trigger dbo.PIMS_RSCNOT_A_S_IUD_TR'
+GO
+DROP TRIGGER IF EXISTS [dbo].[PIMS_RSCNOT_A_S_IUD_TR]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop index dbo.RSCNOT_RESEARCH_FILE_ID_IDX
+PRINT N'Drop index dbo.RSCNOT_RESEARCH_FILE_ID_IDX'
+GO
+DROP INDEX IF EXISTS [dbo].[PIMS_RESEARCH_FILE_NOTE].[RSCNOT_RESEARCH_FILE_ID_IDX]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop index dbo.RSCNOT_NOTE_ID_IDX
+PRINT N'Drop index dbo.RSCNOT_NOTE_ID_IDX'
+GO
+DROP INDEX IF EXISTS [dbo].[PIMS_RESEARCH_FILE_NOTE].[RSCNOT_NOTE_ID_IDX]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop unique constraint dbo.PIMS_RSCDOC_H_UK
+PRINT N'Drop unique constraint dbo.PIMS_RSCDOC_H_UK'
+GO
+ALTER TABLE [dbo].[PIMS_RESEARCH_FILE_DOCUMENT_HIST]
+	DROP CONSTRAINT IF EXISTS [PIMS_RSCDOC_H_UK]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop primary key constraint dbo.PIMS_RSCDOC_H_PK
+PRINT N'Drop primary key constraint dbo.PIMS_RSCDOC_H_PK'
+GO
+ALTER TABLE [dbo].[PIMS_RESEARCH_FILE_DOCUMENT_HIST]
+	DROP CONSTRAINT IF EXISTS [PIMS_RSCDOC_H_PK]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop unique constraint dbo.PIMS_RSCNOT_H_UK
+PRINT N'Drop unique constraint dbo.PIMS_RSCNOT_H_UK'
+GO
+ALTER TABLE [dbo].[PIMS_RESEARCH_FILE_NOTE_HIST]
+	DROP CONSTRAINT IF EXISTS [PIMS_RSCNOT_H_UK]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop primary key constraint dbo.PIMS_RSCNOT_H_PK
+PRINT N'Drop primary key constraint dbo.PIMS_RSCNOT_H_PK'
+GO
+ALTER TABLE [dbo].[PIMS_RESEARCH_FILE_NOTE_HIST]
+	DROP CONSTRAINT IF EXISTS [PIMS_RSCNOT_H_PK]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop unique constraint dbo.RSCNOT_RSRCH_FILE_NOTE_TUC
+PRINT N'Drop unique constraint dbo.RSCNOT_RSRCH_FILE_NOTE_TUC'
+GO
+ALTER TABLE [dbo].[PIMS_RESEARCH_FILE_NOTE]
+	DROP CONSTRAINT IF EXISTS [RSCNOT_RSRCH_FILE_NOTE_TUC]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop primary key constraint dbo.RSCNOT_PK
+PRINT N'Drop primary key constraint dbo.RSCNOT_PK'
+GO
+ALTER TABLE [dbo].[PIMS_RESEARCH_FILE_NOTE]
+	DROP CONSTRAINT IF EXISTS [RSCNOT_PK]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop unique constraint dbo.RSCDOC_RESEARCH_FILE_DOCUMENT_TUC
+PRINT N'Drop unique constraint dbo.RSCDOC_RESEARCH_FILE_DOCUMENT_TUC'
+GO
+ALTER TABLE [dbo].[PIMS_RESEARCH_FILE_DOCUMENT]
+	DROP CONSTRAINT IF EXISTS [RSCDOC_RESEARCH_FILE_DOCUMENT_TUC]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop primary key constraint dbo.RSCDOC_PK
+PRINT N'Drop primary key constraint dbo.RSCDOC_PK'
+GO
+ALTER TABLE [dbo].[PIMS_RESEARCH_FILE_DOCUMENT]
+	DROP CONSTRAINT IF EXISTS [RSCDOC_PK]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter table dbo.PIMS_RESEARCH_FILE_DOCUMENT
+PRINT N'Alter table dbo.PIMS_RESEARCH_FILE_DOCUMENT'
+GO
+ALTER TABLE [dbo].[PIMS_RESEARCH_FILE_DOCUMENT] DROP CONSTRAINT IF EXISTS [RSCDOC_RESEARCH_FILE_DOCUMENT_ID_DEF]
```

Comment:
@dfilteau can you explain the change from RSCDOC to RFLDOC? I want to make sure this is intentional.